api/server.js

```diff
@@ -0,0 +1,31 @@
+const express = require("express");
+
+const server = express();
+
+server.use(express.json());
+
+const users = [];
+
+server.get("/users", (req, res) => {
+  res.status(200).json(users);
+});
```

Comment:
Nice and easy endpoint to verify that data has been posted and deleted successfully.